octobot/configuration_manager.py

```diff
@@ -118,3 +126,35 @@ def init_config(
         shutil.copyfile(from_config_file, config_file)
     except Exception as global_exception:
         raise Exception(f"Can't init config file {global_exception}")
+
+
+def init_default_profile():
+    profile_folder = os.path.join(common_constants.USER_PROFILES_FOLDER,
+                                  common_constants.DEFAULT_PROFILE)
+    if not os.path.exists(profile_folder):
+        os.makedirs(profile_folder)
+    shutil.copyfile(constants.DEFAULT_PROFILE_FILE,
+                    os.path.join(profile_folder, common_constants.DEFAULT_PROFILE_FILE))
+    shutil.copyfile(constants.DEFAULT_PROFILE_AVATAR,
+                    os.path.join(profile_folder, constants.DEFAULT_PROFILE_AVATAR_FILE_NAME))
+    if os.path.isdir(tentacles_manager_constants.USER_REFERENCE_TENTACLE_CONFIG_PATH):
+        shutil.copyfile(tentacles_manager_constants.USER_REFERENCE_TENTACLE_CONFIG_FILE_PATH,
+                        os.path.join(profile_folder, tentacles_manager_constants.constants.CONFIG_TENTACLES_FILE))
+        shutil.copytree(tentacles_manager_constants.USER_REFERENCE_TENTACLE_SPECIFIC_CONFIG_PATH,
+                        os.path.join(profile_folder, tentacles_manager_constants.TENTACLES_SPECIFIC_CONFIG_FOLDER))
+
+
+def migrate_from_previous_config(config):
+    if common_constants.CONFIG_CRYPTO_CURRENCIES in config.config:
+        # config migration required
+        # add missing exchange enabled config
+        for exchange_config in config.config[common_constants.CONFIG_EXCHANGES].values():
+            exchange_config[common_constants.CONFIG_ENABLED_OPTION] = \
+                exchange_config.get(common_constants.CONFIG_ENABLED_OPTION, True)
+        for key in ("tentacles-packages", "performance-analyser", "PERF", "SAVE_EVALUATIONS"):
```

Comment:
Shouldn't it be a constant in constants ? To be more accessible

Reply:
I removed the from constants to that we know that we don't use them anymore, I think it's better to leave it this way and just remove them when we will need to updater the configuration migrator in later versions

Reply:
ok